bin/config/Config.ts

```diff
@@ -0,0 +1,49 @@
+import commander from 'commander'
+import Configstore from 'configstore'
+import { actionRunner, pkLogger, PKMessageType } from '../polykey'
```

Comment:
Can we use `@polykey`?

Reply:
I didn't think you could do that outside the `src` directory, it throws an error for me.

Reply:
I've added `@` relative imports to the src directory.

Reply:
Yeah I think we wouldn't be able to have relative imports outside of `src` unless we change our baseUrl.